website_v2/gcloud-powershell-website-v2/www/static/app/app.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+    var app = angular.module('powershellSite', ['ngRoute']);
+
+    app.config(function($routeProvider, $locationProvider) {
+        $routeProvider
+            .when('/', {
+                templateUrl: 'static/templates/home.html',
+                controller: 'ReferenceCtrl',
+                controllerAs: 'ref',
```

Comment:
I would do  controllerAs: 'refCtrl'  It will make things much more clear in the ng templates